javascriptv3/example_code/dynamodb/tests/ddbdoc_delete_item.test.js

```diff
@@ -0,0 +1,20 @@
+import { mockClient } from "aws-sdk-client-mock";
+import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
+
+const ddbMock = mockClient(DynamoDBDocumentClient);
+
+beforeEach(() => {
+    ddbMock.reset();
+});
+
+import 'regenerator-runtime/runtime'
+import { run } from "../src/ddbdoc_delete_item";
+import { DeleteCommand } from "@aws-sdk/lib-dynamodb";
+
+it("should get user names from the DynamoDB", async () => {
```

Comment:
It looks like this description is wrong for this and most following tests (cut and paste error).

Reply:
updated